lib/v2/screens/dashboard/interactor/viewmodels/token_balance_view_model.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:seeds/v2/datasource/remote/model/balance_model.dart';
+import 'package:seeds/v2/datasource/remote/model/token_model.dart';
+import 'package:seeds/v2/utils/double_extension.dart';
+
+class TokenBalanceViewModel {
+  final TokenModel token;
+  final BalanceModel? balance;
+  final bool errorLoading;
```

Comment:
Another attempt to have the model hold responsibility loading error or success. 
Models should be very very dumb, all they do is hold.

Reply:
It's a view model - therefore it has to model a loading error vs loading success, no?

Reply:
Since the view needs to potentially show that there was a load error

Reply:
yes, my bad, didnt notice this is a view model and not a model